lib/scala/syntax/definition/src/main/scala/org/enso/syntax/text/ast/meta/Builtin.scala

```diff
@@ -267,16 +269,57 @@ object Builtin {
     }
 
     val `import` = {
+      val qualNamePat     = Pattern.SepList(Pattern.Cons(), Opr("."))
+      val rename          = Var("as") :: Pattern.Cons()
+      val all: Pattern    = Var("all")
```

Comment:
The `all` pattern seems to not be described in the [imports syntax documentation](https://github.com/enso-org/enso/blob/wip/mk/import-syntax/docs/syntax/imports.md).

Reply:
None of this whole bunch of work is described. I'm going to update the docs as the last PR in this series.